templates/contribute/pr-review.md

```diff
@@ -0,0 +1,723 @@
+everyone is welcome, and even encouraged, to *review* pull requests. (Note: there is in 
+fact another category of people, mathlib *reviewers*, who have shown that they can provide
+valuable PR reviews; approving reviews from these users with `maintainer merge` are more
+quickly merged by maintainers).
+
+A history of helpful reviews is a key criterion for membership on either the mathlib
+reviewer or mathlib maintainer teams.
+
+This guide starts with general [guidelines](#guidelines-for-review) for conducting reviews,
+a high-level [overview](#what-to-consider-when-reviewing) of what reviewers should be
+considering and then focuses on a few practical [examples](#examples).
+
+While this guide is rather long, it is not required to comprehend everything before
+starting to review. Partial reviews are themselves helpful, and you can learn about the
+various considerations in a piecemeal fashion as you progress in your understanding
+of Lean and mathlib.
+
+## Guidelines for review
+
+### Respect and encouragment
+
+As with all interactions in the mathlib community, be sure to adhere to the
+[Code of Conduct]. In short, be respectful. However, when reviewing please be
+sure to also be *encouraging*. The majority of contributors have only made
+a handful of pull requests, and this may even be their first! As such it is
+important to avoid comments like "This result is useless, we already have a
+version of it." Instead you could be more gentle and say, for example, "Thanks
+for proving this, but I think we already have a lemma to this effect. It is
+`my_generic_lemma`. Please try to use that instead." Try to find the
+good in what they have done, even while pointing out room for improvement.
+
+### Humility
+
+None of us, including the mathlib maintainers, is perfect or has a monopoly
+on best practices. As such, reviewers should always leave room for someone
+to suggest a better approach. In addition, it's important to recognize that
+*you may be wrong*, and to allow for this possibility. Of course, perfect is
+the enemy of the good, so it is not necessary to wait indefinitely for 
+better approaches or new ideas.
+
+## What to consider when reviewing
+
+Reviewing is essentially about looking at code and asking yourself questions.
+The foundational issues, organized approximately from easiest to hardest, are:
+style, documentation, location, improvements and library integration.
+
+Note that new reviewers are certainly capable of commenting on the first two
+or three issues, whereas answering questions of library integration generally
+requires several months of developing familiarity with mathlib and contributing
+toward the reviewing process.
+
+Here are some explicit questions you can ask yourself as a reviewer.
+
+- [does it adhere to style?](#style)
+    + [code formatting](style.html)
+    + [naming conventions](naming.html)
+    + is the [PR title](link-to-commit-conventions) and description appropriately informative?
```

Comment:
Just to note that there are still some invalid links in here like this one.

Reply:
oops thanks, will fix!